### ISSUE TITLE
BlueSlimeCore update breaks CombatLogX

### ISSUE THREAD

**Issue by SlimeDog** [SirBlobman/CombatLogX#777]
### Server Implementation

Paper

### Server Version

other

### Describe the bug

BlueSlimeCore update breaks CombatLogX

### Reproduction Steps

- Update BlueSlimeCore
- Start server
- Read errors in log

### Expected behaviour

Clean start-up

### latest log file (if applicable)

See below

### CombatLogX Version

Cannot; CLX is disabled.

Paper 1.20.4-324
BlueSlimeCore version 2.9.4.Beta-400
CombatLogX 11.4.0.2.Beta-1222

### Anything else?

```
[22:17:07] [Server thread/INFO]: [CombatLogX] Enabling CombatLogX v11.4.0.2.Beta-1222
[22:17:07] [Server thread/INFO]: [CombatLogX] Successfully loaded 1 language(s).
[22:17:07] [Server thread/INFO]: [CombatLogX] Detected server as regular SpigotMC/PaperMC (not Folia)
[22:17:07] [Server thread/ERROR]: Error occurred while enabling CombatLogX v11.4.0.2.Beta-1222 (Is it up to date?)
java.lang.ExceptionInInitializerError: null
        at com.github.sirblobman.api.shaded.adventure.text.serializer.gson.GsonComponentSerializerImpl$BuilderImpl.<init>(GsonComponentSerializerImpl.java:125) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.text.serializer.gson.GsonComponentSerializer.builder(GsonComponentSerializer.java:84) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.platform.bukkit.BukkitComponentSerializer.<clinit>(BukkitComponentSerializer.java:67) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.platform.bukkit.BukkitFacet$Message.createMessage(BukkitFacet.java:74) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.platform.bukkit.BukkitFacet$Message.createMessage(BukkitFacet.java:67) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.platform.facet.FacetAudience.createMessage(FacetAudience.java:521) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.platform.facet.FacetAudience.sendMessage(FacetAudience.java:191) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.audience.Audience.sendMessage(Audience.java:250) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.audience.Audience.sendMessage(Audience.java:214) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.audience.ForwardingAudience.sendMessage(ForwardingAudience.java:104) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.language.LanguageManager.sendMessage(LanguageManager.java:701) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.language.LanguageManager.sendMessage(LanguageManager.java:687) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.language.LanguageManager.broadcastMessage(LanguageManager.java:841) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.language.LanguageManager.broadcastMessage(LanguageManager.java:835) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.combatlogx.CombatPlugin.broadcastMessageOnLoad(CombatPlugin.java:305) ~[CombatLogX-11.4.0.2-b1222.jar:?]
        at com.github.sirblobman.combatlogx.CombatPlugin.onEnable(CombatPlugin.java:110) ~[CombatLogX-11.4.0.2-b1222.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:190) ~[paper-1.20.4.jar:git-Paper-342]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[paper-1.20.4.jar:git-Paper-342]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_20_R3.CraftServer.enablePlugin(CraftServer.java:637) ~[paper-1.20.4.jar:git-Paper-342]
        at org.bukkit.craftbukkit.v1_20_R3.CraftServer.enablePlugins(CraftServer.java:548) ~[paper-1.20.4.jar:git-Paper-342]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:673) ~[paper-1.20.4.jar:git-Paper-342]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:433) ~[paper-1.20.4.jar:git-Paper-342]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:308) ~[paper-1.20.4.jar:git-Paper-342]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1132) ~[paper-1.20.4.jar:git-Paper-342]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:321) ~[paper-1.20.4.jar:git-Paper-342]
        at java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.IllegalStateException: Key adventure:json/emit/rgb has already been used. Option keys must be unique.
        at net.kyori.option.OptionImpl.option(OptionImpl.java:49) ~[option-1.0.0.jar:?]
        at net.kyori.option.Option.booleanOption(Option.java:52) ~[option-1.0.0.jar:?]
        at com.github.sirblobman.api.shaded.adventure.text.serializer.json.JSONOptions.<clinit>(JSONOptions.java:53) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        ... 28 more
[22:17:07] [Server thread/INFO]: [CombatLogX] Disabling CombatLogX v11.4.0.2.Beta-1222
[22:17:07] [Server thread/INFO]: [CombatLogX] Disabling expansions...
[22:17:07] [Server thread/INFO]: [CombatLogX] There were no expansions to disable.
[22:17:07] [Server thread/INFO]: [CombatLogX] Successfully disabled all expansions.
[22:17:07] [Server thread/ERROR]: Error occurred while disabling CombatLogX v11.4.0.2.Beta-1222
java.lang.NoClassDefFoundError: Could not initialize class com.github.sirblobman.api.shaded.adventure.platform.bukkit.BukkitComponentSerializer
        at com.github.sirblobman.api.shaded.adventure.platform.bukkit.BukkitFacet$Message.createMessage(BukkitFacet.java:74) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.platform.bukkit.BukkitFacet$Message.createMessage(BukkitFacet.java:67) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.platform.facet.FacetAudience.createMessage(FacetAudience.java:521) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.platform.facet.FacetAudience.sendMessage(FacetAudience.java:191) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.audience.Audience.sendMessage(Audience.java:250) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.audience.Audience.sendMessage(Audience.java:214) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.audience.ForwardingAudience.sendMessage(ForwardingAudience.java:104) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.language.LanguageManager.sendMessage(LanguageManager.java:701) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.language.LanguageManager.sendMessage(LanguageManager.java:687) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.language.LanguageManager.broadcastMessage(LanguageManager.java:841) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.language.LanguageManager.broadcastMessage(LanguageManager.java:835) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.combatlogx.CombatPlugin.broadcastMessageOnDisable(CombatPlugin.java:327) ~[CombatLogX-11.4.0.2-b1222.jar:?]
        at com.github.sirblobman.combatlogx.CombatPlugin.onDisable(CombatPlugin.java:130) ~[CombatLogX-11.4.0.2-b1222.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:283) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.disablePlugin(PaperPluginInstanceManager.java:225) ~[paper-1.20.4.jar:git-Paper-342]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.disablePlugin(PaperPluginManagerImpl.java:109) ~[paper-1.20.4.jar:git-Paper-342]
        at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:537) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:194) ~[paper-1.20.4.jar:git-Paper-342]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[paper-1.20.4.jar:git-Paper-342]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_20_R3.CraftServer.enablePlugin(CraftServer.java:637) ~[paper-1.20.4.jar:git-Paper-342]
        at org.bukkit.craftbukkit.v1_20_R3.CraftServer.enablePlugins(CraftServer.java:548) ~[paper-1.20.4.jar:git-Paper-342]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:673) ~[paper-1.20.4.jar:git-Paper-342]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:433) ~[paper-1.20.4.jar:git-Paper-342]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:308) ~[paper-1.20.4.jar:git-Paper-342]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1132) ~[paper-1.20.4.jar:git-Paper-342]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:321) ~[paper-1.20.4.jar:git-Paper-342]
        at java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.ExceptionInInitializerError [in thread "Server thread"]
        at com.github.sirblobman.api.shaded.adventure.text.serializer.gson.GsonComponentSerializerImpl$BuilderImpl.<init>(GsonComponentSerializerImpl.java:125) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.text.serializer.gson.GsonComponentSerializer.builder(GsonComponentSerializer.java:84) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.platform.bukkit.BukkitComponentSerializer.<clinit>(BukkitComponentSerializer.java:67) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.platform.bukkit.BukkitFacet$Message.createMessage(BukkitFacet.java:74) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.platform.bukkit.BukkitFacet$Message.createMessage(BukkitFacet.java:67) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.platform.facet.FacetAudience.createMessage(FacetAudience.java:521) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.platform.facet.FacetAudience.sendMessage(FacetAudience.java:191) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.audience.Audience.sendMessage(Audience.java:250) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.audience.Audience.sendMessage(Audience.java:214) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.shaded.adventure.audience.ForwardingAudience.sendMessage(ForwardingAudience.java:104) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.language.LanguageManager.sendMessage(LanguageManager.java:701) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.language.LanguageManager.sendMessage(LanguageManager.java:687) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.language.LanguageManager.broadcastMessage(LanguageManager.java:841) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.api.language.LanguageManager.broadcastMessage(LanguageManager.java:835) ~[BlueSlimeCore-2.9.4-b400.jar:?]
        at com.github.sirblobman.combatlogx.CombatPlugin.broadcastMessageOnLoad(CombatPlugin.java:305) ~[CombatLogX-11.4.0.2-b1222.jar:?]
        at com.github.sirblobman.combatlogx.CombatPlugin.onEnable(CombatPlugin.java:110) ~[CombatLogX-11.4.0.2-b1222.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281) ~[paper-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:190) ~[paper-1.20.4.jar:git-Paper-342]
        ... 10 more
```

**Comment by SirBlobman** [comment]
Thank you for reporting this issue. Its theoretically fixed but I need to do more testing. The CombatLogX auto-build should be out soon.

**Comment by SlimeDog** [comment]
Seems to be fixed with BlueSlimeCore 2.9.4.402 and CombatLogX 11.4.0.2-b1225. Thanks.
